trackier-ios-sdk/Classes/TrackierSDK.swift

```diff
@@ -71,4 +71,12 @@ public class TrackierSDK {
     public static func trackAsOrganic(organic: Bool) {
         shared.instance.organic = organic
     }
+    
+    public static func setUserPhone(userPhone: String){
```

Comment:
please note the space after the ) in the above functions @IloveJavaa 
```suggestion
    public static func setUserPhone(userPhone: String) {
```

Reply:
Changes done
<img width="481" alt="Screenshot 2022-08-08 at 11 24 46 AM" src="https://user-images.githubusercontent.com/16884982/183348856-b48eb3a1-c1d8-4e16-9df0-69eca77eb0e0.png">